src/Datadog.Trace.ClrProfiler.Managed/Integrations/Testing/NUnitIntegration.cs

```diff
@@ -342,20 +183,148 @@ private static void FinishScope(Scope scope, object testExecutionContext, Except
             }
 
             execute(workShift);
-            SynchronizationContext context = SynchronizationContext.Current;
+            AutoInstrumentation.Testing.Common.FlushSpans();
```

Comment:
Should this use the FlushSpans overload that takes an integration?

Reply:
Yes, fixed in 45506f2